tests/Containers/MerkleTreeTest.cpp

```diff
@@ -505,14 +470,166 @@ TEST(MerkleTreeTest, test_serializeBinary) {
 
   std::string t1s;
   t1.serializeBinary(t1s, true);
+  ASSERT_EQ(520, t1s.size());
+
+  std::unique_ptr<::arangodb::containers::MerkleTree<::arangodb::containers::FnvHashProvider, 3>> t2 =
+      ::arangodb::containers::MerkleTree<::arangodb::containers::FnvHashProvider, 3>::fromBuffer(t1s);
+  ASSERT_NE(t2, nullptr);
+  ASSERT_TRUE(t1.diff(*t2).empty());
+  ASSERT_TRUE(t2->diff(t1).empty());
```

Comment:
In such cases I prefer `EXPECT_xxx` instead of `ASSERT_xxx` as that way we have more information if the test fails (but no need to change this now).

Reply:
Ok, then for simplicity we will leave it as is here :sweat_smile: